Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -39,17 +39,25 @@ function Get-AzureUri
 function Get-ProxyUri
 {
     param([String] [Parameter(Mandatory=$true)] $serverUrl)
+    
+    $proxyUri = (Get-Item env:AGENT_PROXYURL).value
```

Comment:
this will blow up if the env var isnt set. $env:AGENT_PROXYURL would be better. or even get-vstsvariable which adds tracing.